# Asyncio Evolved: Enhanced Exception Handling with TaskGroup in Python 3.11

This code is to accompany a talk.

[Asyncio Evolved: Enhanced Exception Handling with Python 3.11 TaskGroups - PyCon TW 2023](https://tw.pycon.org/2023/en-us/conference/talk/280)

# Operation check

This code has been tested and confirmed to work with the following versions:

* Python 3.10.11
    * [Python Release Python 3.10.11 \| Python.org](https://www.python.org/downloads/release/python-31011/)

* Python 3.11.3
    * [Python Release Python 3.11.3 \| Python.org](https://www.python.org/downloads/release/python-3113)

